rev1

keremgocen opened the pull request:
- add go app to pull data from firebase periodically
- docker-compose brings up influx and grafana to display metrics
- work in progress

jacekelgda commented on mrmoody-metrics/Dockerfile:
```diff
@@ -0,0 +1,5 @@
+CMD ["apt-get install -y ca-certificates"]
+ADD mrmoody /app
+ADD config.json /
+CMD ["./app"]
```
pls fix

jacekelgda commented on mrmoody-metrics/config.json:
```diff
@@ -0,0 +1,10 @@
+    "db_name": "",
+    "db_user": "",
+    "db_pass": ""
+}
```
pls fix

jacekelgda commented on mrmoody-metrics/docker-compose.yml:
```diff
@@ -0,0 +1,18 @@
+    ports:
+     - "3000:3000"
+    links:
+     - influx
```
pls fix

jacekelgda commented on mrmoody-metrics/README.md:
```diff
@@ -0,0 +1,61 @@
+### Metrics for Mr.Moody
+
+### Install Go
+https://golang.org/doc/install
```
So, the only way to run it is to install go on host machine ? Or is there an alternate way of running this using docker ?

jacekelgda commented on mrmoody-metrics/config.json:
```diff
@@ -0,0 +1,10 @@
+{
```
if this is a template file, maybe it's better to save it as .dist or .template and the add the actual file to .gitignore ?

jacekelgda commented on mrmoody-metrics/docker-compose.yml:
```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  web:
+    image: "moody"
```
this is an error - you probably have image moody build locally, that's why it works

jacekelgda commented on mrmoody-metrics/main.go:
```diff
@@ -0,0 +1,147 @@
+	}
+	req.Header.Add("Authentication", key)
+
+	// now := time.Now()
```
please remove all unused code

jacekelgda commented on mrmoody-metrics/main.go:
```diff
@@ -0,0 +1,147 @@
+	// log.Println("last month:", lastMonth, " ts:", lastMonthStr)
+
+	q := req.URL.Query()
+	// q.Add("orderBy", "\"date\"")
```
please remove all unused code

jacekelgda commented on mrmoody-metrics/storage/vault101.go:
```diff
@@ -0,0 +1,106 @@
+	// namely INFLUX_USER/INFLUX_PWD. If not just omit Username/Password below.
+	c, err := client.NewHTTPClient(client.HTTPConfig{
+		Addr: addr,
+		// Username: os.Getenv("INFLUX_USER"),
```
please remove all unused code

Commit:
rev1

- add go app to pull data from firebase periodically
- docker-compose brings up influx and grafana to display metrics
- work in progress

jacekelgda committed:
Docker fixes

keremgocen commented:
Closing this to make a new PR with Docker setup and docs only.
